File: app/src/main/java/com/alaaeltaweel/thikrallah/RadioActivity.java
package com.alaaeltaweel.thikrallah

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import com.alaaeltaweel.thikrallah.presentation.screen.radio.RadioScreen

class RadioActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            RadioScreen(onBackClick = { finish() })
        }
    }
}

File: app/src/main/java/com/alaaeltaweel/thikrallah/RadioActivity.kt
package com.alaaeltaweel.thikrallah

import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.appcompat.app.AppCompatActivity
import com.alaaeltaweel.thikrallah.presentation.screen.radio.RadioScreen

class RadioActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            RadioScreen(onBackClick = { finish() })
        }
    }
}
